typings/react-native-code-push.d.ts

```diff
@@ -146,77 +146,99 @@ export interface SyncOptions {
      */
     updateDialog?: UpdateDialog;
 }
-    
+
 export interface UpdateDialog {
     /**
      * Indicates whether you would like to append the description of an available release to the
      * notification message which is displayed to the end user. Defaults to false.
      */
     appendReleaseDescription?: boolean;
-    
+
     /**
      * Indicates the string you would like to prefix the release description with, if any, when
      * displaying the update notification to the end user. Defaults to " Description: "
      */
     descriptionPrefix?: string;
-    
+
     /**
      * The text to use for the button the end user must press in order to install a mandatory update. Defaults to "Continue".
      */
     mandatoryContinueButtonLabel?: string;
-    
+
     /**
      * The text used as the body of an update notification, when the update is specified as mandatory.
      * Defaults to "An update is available that must be installed.".
      */
     mandatoryUpdateMessage?: string;
-    
+
     /**
      * The text to use for the button the end user can press in order to ignore an optional update that is available. Defaults to "Ignore".
      */
     optionalIgnoreButtonLabel?: string;
-    
+
     /**
      * The text to use for the button the end user can press in order to install an optional update. Defaults to "Install".
      */
     optionalInstallButtonLabel?: string;
-    
+
     /**
      * The text used as the body of an update notification, when the update is optional. Defaults to "An update is available. Would you like to install it?".
      */
     optionalUpdateMessage?: string;
-    
+
     /**
      * The text used as the header of an update notification that is displayed to the end user. Defaults to "Update available".
      */
     title?: string;
 }
-      
-declare namespace CodePush { 
+
+export interface StatusReport {
+    /**
+     * Whether the deployment succeeded or failed.
+     */
+    status: CodePush.DeploymentStatus;
+
+    /**
+     * Details of the package that was deployed (or attempted to).
+     */
+    package: Package;
+
+    /**
+     * Deployment key used when deploying the previous package.
+     */
+    previousDeploymentKey?: string;
+
+    /**
+     * The label (v#) of the package that was upgraded from.
+     */
+    previousLabelOrAppVersion?: string;
```

Comment:
There is also an `appVersion` key in the report, we also report binary updates. For more info: https://github.com/Microsoft/react-native-code-push/blob/master/ios/CodePush/CodePushTelemetryManager.m#L29